ci/build_cpp.sh

```diff
@@ -19,9 +19,24 @@ rapids-logger "Begin C++ and Python builds"
 
 sccache --zero-stats
 
-RAPIDS_PACKAGE_VERSION=$(rapids-generate-version) rapids-conda-retry mambabuild \
-  conda/recipes/ucxx
+RAPIDS_PACKAGE_VERSION=$(rapids-generate-version)
+export RAPIDS_PACKAGE_VERSION
+
+source rapids-rattler-channel-string
+
+# --no-build-id allows for caching with `sccache`
+# more info is available at
+# https://rattler.build/latest/tips_and_tricks/#using-sccache-or-ccache-with-rattler-build
+rattler-build build --recipe conda/recipes/ucxx \
+                    --experimental \
+                    --no-build-id \
+                    --channel-priority disabled \
+                    --output-dir "$RAPIDS_CONDA_BLD_OUTPUT_DIR" \
+                    "${RATTLER_CHANNELS[@]}"
 
 sccache --show-adv-stats
 
+# remove build_cache directory
+rm -rf "$RAPIDS_CONDA_BLD_OUTPUT_DIR"/build_cache
```

Comment:
This seems like something that should be an ignore pattern in our upload script instead of something that every build script has to include.

Reply:
Agreed. We opened an issue upstream, too, to add a CLI option to clear the build-cache post-build: https://github.com/prefix-dev/rattler-build/issues/1424